.github/workflows/build-and-publish-images.yml

```diff
@@ -46,6 +46,7 @@ jobs:
       PYTHON_VER: ${{ matrix.PYTHON_VER }}
       IMAGE_REPO: ${{ matrix.IMAGE_REPO }}
       IMAGE_NAME: ${{ matrix.IMAGE_NAME }}
+      MANYLINUX_VER: ${{ matrix.MANYLINUX_VER }}
```

Comment:
`MANYLINUX_VER` looks to be a build argument that is computed in `ci/compute-build-args.sh`.

Therefore I don't think it needs to be computed at the top-level and passed around as a workflow argument.